app/assets/javascripts/workflows.js.coffee

```diff
@@ -161,7 +161,7 @@ $(window).focus ->
     $("#terminal_button").bind('click', false)
 
 @update_submit_button = (id, status_char) ->
-  if id? && !status_char?
+  if id? && status_char != "R" && status_char != "Q"
```

Comment:
In active states between not submitted and completed/failed are many. Running and Queued and Suspended and Held etc. Better to enable if completed/failed or not submitted, instead of trying to match against an active state here.

```
&& ((!status_char?) || status_char == "C" || status_char == "F")
```

or this might be clearer

```
if id? && (status_char in [null, "C", "F"])
```

or you write a function `status_not_active(status_char)` that does this.